clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java

```diff
@@ -130,13 +145,16 @@ private void send(
 
   @Override
   public ActivateJobsCommandStep3 tenantId(final String tenantId) {
-    builder.addTenantIds(tenantId);
+    areCustomTenantIdsSet = true;
+    customTenantIds.add(tenantId);
     return this;
   }
 
   @Override
   public ActivateJobsCommandStep3 tenantIds(final List<String> tenantIds) {
-    builder.addAllTenantIds(tenantIds);
+    areCustomTenantIdsSet = true;
+    customTenantIds.clear();
```

Comment:
🔧 If you don't clear the `customTenantIds` here, then users can chain `tenantIds` calls just like `tenantId` calls. They can also combine it with `tenantId` then.